### ISSUE TITLE
Localized compactness factor failing in parallel

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1606]
cq-id: VisIt00007930
cq-submitter: Hank Childs
cq-submit-date: 03/27/07

Testing every VisIt query in parallel with a sil selection that left processors with no data, the localized compactness factor query caused the engine to crash.  So we should fix up this query.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Cyrus Harrison
Original creation: 06/21/2010 07:13 pm
Original update: 06/21/2010 07:13 pm
Ticket number: 19
